src/appMain/sdl_preloaded_pt.json

```diff
@@ -3403,7 +3407,44 @@
                     "name": "prndl",
                     "key": "OEM_REF_PRNDL",
                     "type": "PRNDL",
-                    "mandatory": false
+                    "mandatory": false,
+                    "since": "1.0",
+                    "until": "6.2"
+                },
+                {
```

Comment:
Could you please explain why two `prndl` vehicle data items are required.

Reply:
@ShobhitAd I believe that we do not need the second one, removed in 89b14da7ace456037df1e73f9d262d18d850e573